spec/features/dsl/scheduled_transfers_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+RSpec.describe 'DSL - Scheduled Transfer' do
+  let(:client)                { setup_client }
+  let(:client_id)             { 'client-id' }
+  let(:client_secret)         { 'client-secret' }
+
+  before do
+    client.config.environment = FidorApi::Environment::Future.new
+  end
+
+  describe '#scheduled_transfers' do
+    before do
+      stub_fetch_request(
+        endpoint:      %r{/scheduled_transfers},
+        response_body: [
+          {
+            'id':                    'c51038a3-8ace-4e56-9009-72a5d319b66b',
+            'account_id':            '12345678',
+            'external_uid':          '4279762F8',
+            'amount':                4223,
+            'currency':              'EUR',
+            'subject':               'Money for you',
+            'status':                'created',
+            'additional_attributes': {},
+            'beneficiary':           {
+                                      'unique_name':  'string',
```

Comment:
Let's don't introduce this style of indentation for nested hashes.
Please check the other specs for examples.

Preferred style:

```ruby
hello: 42,
world: {
  foo: 'bar',
  blah: {
    blubb: 23
  }
}
```

Reply:
took Jobst and me 15 mins to make rubocop pass in the previous way to solve it ...... (cry) (cry) back to square root 1

Reply:
If this is the case we might need to tweak rubocop config here.